### PR TITLE
Add Beiselen DigiSilo Radar Sensor

### PR DESCRIPTION
**Pull request by diplfranzhoepfinger**
Beiselen DigiSilo Radar Sensor

#### Summary

Beiselen DigiSilo Radar Sensor

#### Changes

Beiselen DigiSilo Radar Sensor

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/index.yaml [hunk 1]
```diff
@@ -1551,3 +1551,14 @@ vendors:
 
   - id: the-box-development
     name: The-Box Development
+
+  - # Unique identifier of the vendor (lowercase, alphanumeric with dashes, max 36 characters)
+    id: beiselen
+    # Vendor company name
+    name: Beiselen Service GmbH
+    # LoRa Alliance issued Vendor ID
+    vendorID: 845
```
Comment (johanstokking) [inline vendor/index.yaml:1560]: Did you guys join the LoRa Alliance recently? I don't see you in the new members list.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,9 @@ node_modules
 .DS_Store
 bin/runscript
 .envrc
+
+libiconv2.dll
+
+libintl3.dll
+
+make.exe
```
Comment (johanstokking) [inline .gitignore:11]: Please revert this change

**Review by johanstokking — COMMENTED** [review]
Welcome @diplfranzhoepfinger!

Two questions, otherwise looks good!

File: .gitignore (hunk 2 above)
Comment (diplfranzhoepfinger) [reply .gitignore]: done

**Comment by diplfranzhoepfinger** [comment]
all 2 things done. 

LORA Alliance Member Request we will do ASAP !

**Comment by diplfranzhoepfinger** [comment]
should be good now ...

**Review by johanstokking — APPROVED** [review]

**Comment by johanstokking** [comment]
Thanks guys, we welcome further contributions!
